# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wolfssl/libdir.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- wolfssl-5.6.6-stable/CMakeLists.txt.vanilla	2024-02-24 14:28:37.568000000 +0100
+++ wolfssl-5.6.6-stable/CMakeLists.txt	2024-02-24 14:36:43.668000000 +0100
@@ -2530,8 +2530,8 @@
 # Install the library
 install(TARGETS wolfssl
         EXPORT wolfssl-targets
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib
+        LIBRARY DESTINATION lib64
+        ARCHIVE DESTINATION lib64
         RUNTIME DESTINATION bin
         )
 # Install the headers
@@ -2563,7 +2563,7 @@
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
 set(exec_prefix "\${prefix}")
-set(libdir "\${exec_prefix}/lib")
+set(libdir ${CMAKE_INSTALL_LIBDIR})
 set(includedir "\${prefix}/include")
 set(VERSION ${PROJECT_VERSION})
 
